packages/11ty/_plugins/figures/schema.json

```diff
@@ -0,0 +1,175 @@
+{
+  "$id": "https://quire.getty.com/figures.schema.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$ref": "#/definitions/Figures",
+  "title": "Figures",
+  "description": "Quire figures data schema",
+  "type": "object",
+  "definitions": {
+    "Figures": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "figure_list": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/FigureList"
+          }
+        }
+      },
+      "required": [
+        "figure_list"
+      ],
+      "title": "Figures"
+    },
+    "FigureList": {
+      "type": "object",
+      "additionalProperties": false,
```

Comment:
I wonder if we should allow additional properties - this might be a key area where people who are using Quire in an off-menu way might expand on the project. I guess they could go in and edit this schema if they wanted to do that... but just a thought.

Reply:
@hbalenda allowing additional properties is a good idea; initially I think we can set it to `false` and then either update the schema to allow them or perhaps implement this as a Quire project setting that treats additional properties as warnings rather than errors when we are validating the figures data.